### ISSUE TITLE
[BUG] Unter Einstellungen/Akten/Dokumentorder kann ohne Sicherheitsabfrage die Struktur gelöscht werden

### ISSUE THREAD

**Issue by steboe1** [jlawyerorg/j-lawyer-org#2429]
Keine Sicherheitsabfrage beim löschen der Struktur im Editor Dokumentordner 
Die gelöschte Struktur ist im Papierkorb nicht abgelegt

![image](https://github.com/jlawyerorg/j-lawyer-org/assets/168509750/8690d1ca-7c54-483a-a930-640b658f4e7f)

**Comment by librelegal** [comment]
ich stimme zu, auch hier sollte vor dem Löschen eine Warnung erfolgen (vergleichbar bei Akten https://github.com/jlawyerorg/j-lawyer-org/issues/3409)
